### DISCUSSION TITLE
尝试了一下alist用alist V3驱动挂在其他人的alist，但是获取不到内容

### DISCUSSION THREAD

**Discussion by Rhin0cer0s2022** [AlistGo/alist#2895] [Q&A]
尝试了一下alist用alist V3驱动挂在其他人的alist，但是获取不到内容，找了官方文档也没说V3驱动怎么用，所以来此请教

**Comment by NovaShen0724** [comment]
我实测可以使用
挂载路径/test
根文件夹路径 /
链接 https://pan.appek.ml/
其余留空或默认
请测试并回复

**Reply by Rhin0cer0s2022** [reply]
你发的这个确实可以，但是我尝试七米蓝就不行，https://al.chirmyram.com/   这是为什么呢，挂在完成后，列表是空的

**Reply by NovaShen0724** [reply]
请使用v2驱动，目标站点并非v3，(lll￢ω￢)

**Reply by NovaShen0724** [reply]
请提交Q&A以计算contribution

**Reply by Rhin0cer0s2022** [reply]
还真是，非常感谢

**Reply by FEOQin** [reply]
请问使用v2驱动，挂载七米蓝https://al.chirmyram.com/ ，不能索引里面的内容该怎么解决。
